src/main/java/org/alliancegenome/curation_api/services/helpers/UniqueIdentifierHelper.java

```diff
@@ -0,0 +1,76 @@
+package org.alliancegenome.curation_api.services.helpers;
+
+import org.alliancegenome.curation_api.model.entities.Annotation;
+import org.alliancegenome.curation_api.model.entities.base.AuditedObject;
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.model.ingest.dto.AnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.DiseaseAnnotationDTO;
+import org.alliancegenome.curation_api.model.ingest.dto.base.SubmittedObjectDTO;
+import org.apache.commons.lang3.StringUtils;
+
+public class UniqueIdentifierHelper {
+
+	private UniqueIdentifierHelper(){
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.
<!-- __reviewdog__:ChA2MmNiNGQ4YmM5OGU0MTQ0EgpjaGVja3N0eWxl -->